adaptive_navigation/lib/src/adaptive_navigation_scaffold.dart

```diff
@@ -183,15 +180,14 @@ class AdaptiveNavigationScaffold extends StatelessWidget {
     return Drawer(
       child: ListView(
         children: [
-          if (drawerHeader != null) drawerHeader,
+          // TODO: Find a better way to write `drawerHeader!`
+          if (drawerHeader != null) drawerHeader!,
```

Comment:
would `drawerHeader?` work?

Reply:
`?` is used to make types nullable as in `Widget?`, but not used after expressions (as far as I know).

Since `drawerHeader`  is a member variable, type promotion does not work here. Dart SDK does not promote types for member variables, because they could be a `get` function, whose return value can change between calls.

I think `drawerHeader!` is the only way to work around (?) this at the moment.

Reply:
thanks for the explanations!